### DISCUSSION TITLE
Instructor/Student login, Feedback submission

### DISCUSSION THREAD

**Discussion by akshayavenkat2131** [TEAMMATES/teammates#11958] [Help requests] [answered]
Hi team

I have the Front and Back-end server and the datastore emulator running locally.I would like to try out the submitFeedback feature which I am having problems with.I have tried the following : 
- Tried logging in as a student.I seem to be able to login with my registered email address from what I see, but do not see any "Sessions" tab as mentioned in help pages- 

<img width="1270" alt="image" src="https://user-images.githubusercontent.com/68556957/191066738-fa9bb09a-989b-41dc-bf61-31efd93a9094.png">
- Tried submitting a session-link recovery request for which I don't seem to have received any email.
- Tried logging in as an instructor which I am unable to do as well.
- I also did submit a request for an instructor account earlier which I didn't hear back on.

I seem to have hit a roadblock here.Do I need to have any specific accesses to be able to join courses and submit feedback?Would really appreciate some inputs on how to proceed.

Thanks & Regards
Akshaya.

**Comment by zhaojj2209** [comment, marked as answer]
Hello, please refer to the section of our developer documentation on [logging in to TEAMMATES](https://teammates.github.io/teammates/development.html#logging-in-to-a-teammates-instance) for more information! In summary, you will have to log in as administrator first to add instructors and students.
